src/components/CreateProfileForm/CreateProfileFormWrapper.tsx

```diff
@@ -101,6 +101,8 @@ const CreateProfileFormWrapper = ({ children }: PropsWithChildren) => {
       initialValues={initialValues}
       validationSchema={validationSchema}
       onSubmit={handleCreateProfile}
+      validateOnBlur={false}
+      validateOnChange={false}
```

Comment:
**issue:** You disabled live validation completely now - it should validateOnBlur, and validateOnChange but only when form is submitted once - we should use `touched` property to manage that in inputs